pinakes/main/approval/serializers.py

```diff
@@ -121,6 +121,7 @@ class RequestFields:
     FIELDS = (
         "id",
         "requester_name",
+        "group_name",
```

Comment:
@bzwei do we have any specs validating the attribute names returned

Reply:
Added